app/models/map_position.rb

```diff
@@ -6,4 +6,5 @@ class MapPosition < ActiveRecord::Base
 
   has_many :map_locus_hits, foreign_key: 'map_position'
 
+  include Annotable
```

Comment:
This might fail since `map_positions` currently does not have a primary key (or an `id` field).